docusaurus/docs/Android/02-client/02-channels.md

```diff
@@ -657,6 +657,20 @@ channelClient.unmute().enqueue { result ->
 }
 ```
 
+### Checking if user muted a channel
```

Comment:
We use uppercase (well, title case) for headers in the docs

You can use something like https://www.titlecase.com/ to quickly format to that casing